nms/app/packages/magmalte/app/components/context/RefreshContext.js

```diff
@@ -15,12 +15,19 @@
  */
 'use strict';
 import {FetchEnodebs, FetchGateways} from '../../state/lte/EquipmentState';
+import {
+  FetchFegGateways,
+  getActiveFegGatewayId,
+  getFegGatewaysHealthStatus,
+} from '../../state/feg/EquipmentState';
 import {FetchSubscriberState} from '../../state/lte/SubscriberState';
 import {useContext, useEffect, useRef, useState} from 'react';
 import type {EnqueueSnackbarOptions} from 'notistack';
 
 export const REFRESH_INTERVAL = 30000;
 
+export const FEG_GATEWAY = 'feg_gateway';
```

Comment:
If this doesn't exist under MagmaAPIBindings.js, find a better place to put this constant